research/src/pages/breadcrumb.proposal.mdx

```diff
@@ -0,0 +1,66 @@
+---
+menu: Components
+name: Breadcrumb (Editor's Draft)
+path: /components/breadcrumb
+pathToResearch: /components/breadcrumb.research
+---
+
+import BreadcrumbAnatomy from '../components/breadcrumb-anatomy'
+
+## Overview
+
+Breadcrumbs are navigation land marks used to show a user's location within the site or app.
+
+## Use Cases
+
+A `<breadcrumb />` trail consists of a list of links to the parent pages of the current page in hierarchical order. It helps users find their place within a website or web application. Breadcrumbs are often placed horizontally before a page's main content.
+
+## Prior Art/Examples <a href="#prior-art" id="prior-art"></a>
+
+- [Ant Design](https://ant.design/components/breadcrumb/)
+- [Bootstrap](https://getbootstrap.com/docs/4.3/components/breadcrumb/)
+- [Auth0](https://styleguide.auth0.com/components/breadcrumb)
+- [Atlas Kit](https://atlassian.design/components/breadcrumbs/examples)
+- [Carbon](https://www.carbondesignsystem.com/components/breadcrumb/code/)
+- [Fabric](https://developer.microsoft.com/en-us/fluentui#/controls/web/breadcrumb)
+- [Lightning](https://www.lightningdesignsystem.com/components/breadcrumbs/)
+- [Semantic UI](https://semantic-ui.com/collections/breadcrumb.html)
+
+### Anatomy <a class="link" href="#anatomy" id="anatomy"></a>
+
+#### Diagram <a class="link" href="#diagram" id="diagram"></a>
+
+<BreadcrumbAnatomy />
+
+#### Structure <a class="link" href="#structure" id="structure"></a>
+
+- `<breadcrumb>` - The root element that that has the semantic value of navigation **[required]**
+- `<ordered-list>` - The list containing items and dividers **[required]**
+- `<list-item>` - The wrapper to items or divider content **[required]**
+
+Proposed HTML:
+
+```html
+<nav>
+  <ol>
+    <li><a>Item 1</a></li>
+    <li aria-hidden="true">/</li>
+    <li><a>Item 2</a></li>
+  </ol>
+</nav>
+```
+
+#### Accessibility <a class="link" href="#accessibility" id="accessibility"></a>
+
+The Breadcrumb is composed by:
+
+- A `nav` equivalent wrapper that should always have `aria-label` or `aria-labelledby`
+- A ordered `list`
+- `li` items and dividers where the `li` represeting the item can contain anchor or other elements and the `li` represeting the divider should have `aria-hidden="true"`
+- If the last `li` represeting item has an anchor for the current page, this anchor should have `aria-current="page"` otherwise this attribute is optional
```

Comment:
We should note that only anchors which include an href should reference `aria-current="page"`. Anchors without an href are treated as text and should not include `aria-current="page"` as certain AT (such as narrator) will announce the anchor (without an href) as a link anyway.

Reply:
From the [spec](https://www.w3.org/TR/wai-aria-1.2/#aria-current): 
> The aria-current attribute is used when an element within a set of related elements is visually styled to indicate it is the current item in the set...A page token used to indicate a link within a set of pagination links, where the link is visually styled to represent the currently-displayed page.

Anchor without href as text via UIA: https://w3c.github.io/html-aam/#el-a-no-href

Reply:
I would guess that using anchor without `href` is a misusage, not sure if it should be mentioned here.

Reply:
It’s totally valid and treated as text by the browser per the note above. It would be invalid and cause issues when used with aria-current=“page” as that could incorrectly be announced as a link.

Reply:
Isn't this understood by "anchor for the current page"? How could be anchored to the current page without the `href`?

Reply:
Per my note above, an anchor with an href is a link, An anchor without an href is considered text. 

I think the current language in the W3C specification describes this way more succinctly than I could and adopting it in place of the above would resolve this issue.

> The link to the current page has aria-current set to page. If the element representing the current page is not a link, aria-current is optional.

Reply:
Applied